test/powershell/Host/ConsoleHost.Tests.ps1

```diff
@@ -244,17 +244,17 @@ Describe "ConsoleHost unit tests" -tags "Feature" {
 '@
```

Comment:
@iSazonov, your last commit had 1 failures in `PowerShell-CI-macos`
Test-Connection.TraceRoute.returns false without error if MaxHops is exceeded during -Traceroute -Quiet
```powershell
Missing an argument for parameter 'TimeoutSeconds'. Specify a parameter of type 'System.Int32' and try again.
at <ScriptBlock>, /Users/runner/runners/2.163.1/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Management/Test-Connection.Tests.ps1: line 278
278:             Test-Connection 8.8.8.8 -Traceroute -MaxHops 2 -TimeoutSeconds -Quiet | Should -BeFalse

```